packages/core/src/Synchronizer.ts

```diff
@@ -21,6 +21,39 @@ type AttesterSetting = {
     epochLength: number
 }
 
+class BlockQueue {
+    private promises: any[] = []
+    public arr: any[] = []
```

Comment:
I think this can be read-only
so we can use a public getter to access the data
https://github.com/Unirep/Unirep/blob/19e489ccaed0392765654ae743f2c2d04f257358/packages/core/src/Synchronizer.ts#L175-L177

Reply:
Yep, added